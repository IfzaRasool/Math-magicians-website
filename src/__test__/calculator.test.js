import React from 'react';
import renderer from 'react-test-renderer';
import Calculator from '../components/calculator';

it('renders correctly have are no items', () => {
  const calComponent = renderer.create(<Calculator />);
  const tree = calComponent.toJSON();
  expect(tree).toMatchSnapshot();
});
