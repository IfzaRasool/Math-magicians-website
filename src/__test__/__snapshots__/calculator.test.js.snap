// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly have are no items 1`] = `
<div
  className="cal-container"
>
  <p>
    Let us do some Math!!!
  </p>
  <div
    className="calculator"
  >
    <div
      className="cal-screen"
    >
      0
    </div>
    <div
      className="cal-buttons"
    >
      <button
        className="btn"
        onClick={[Function]}
        type="button"
        value="AC"
      >
        AC
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
        value="AC"
      >
        +/-
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
        value="AC"
      >
        %
      </button>
      <button
        className="btn primary"
        onClick={[Function]}
        type="button"
        value="AC"
      >
        รท
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
        value="AC"
      >
        7
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
        value="AC"
      >
        8
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
        value="AC"
      >
        9
      </button>
      <button
        className="btn primary"
        onClick={[Function]}
        type="button"
        value="AC"
      >
        x
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
        value="AC"
      >
        4
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
        value="AC"
      >
        5
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
        value="AC"
      >
        6
      </button>
      <button
        className="btn primary"
        onClick={[Function]}
        type="button"
        value="AC"
      >
        -
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
        value="AC"
      >
        1
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
        value="AC"
      >
        2
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
        value="AC"
      >
        3
      </button>
      <button
        className="btn primary"
        onClick={[Function]}
        type="button"
        value="AC"
      >
        +
      </button>
      <button
        className="btn zero"
        onClick={[Function]}
        type="button"
        value="AC"
      >
        0
      </button>
      <button
        className="btn"
        onClick={[Function]}
        type="button"
        value="AC"
      >
        .
      </button>
      <button
        className="btn primary"
        onClick={[Function]}
        type="button"
        value="AC"
      >
        =
      </button>
    </div>
  </div>
</div>
`;
